libs/labelbox/src/labelbox/schema/member.py

```diff
@@ -0,0 +1,433 @@
+    is_viewer: bool = Field(default=False, frozen=True)
+    is_external_user: bool = Field(default=False, frozen=True)
+    accessible_project_count: Optional[int] = Field(default=None, frozen=True)
+    project_memberships: Set[ProjectMembership] = Field(default=set())
+    can_access_all_projects: bool = Field(default=False)
+    default_role: Optional[Role] = Field(default=None)
+    user_group_ids: Set[str] = Field(default=set())
+    client: Client
+    _current_user_id: str
+
+    def __init__(self, **data):
+        """set private attribute"""
+        super().__init__(**data)
+        self._current_user_id = self.client.get_user().uid
+
+    @field_validator("client", mode="before")
+    @classmethod
+    def experimental(cls, v: Client):
+        if not v.enable_experimental:
+            raise RuntimeError(
+                "Please enable experimental in client to use Members"
+            )
+        return v
+
+    @model_validator(mode="before")
+    def current_user(data: Any) -> Any:
+        if "id" not in data:
+            data["id"] = data["client"].get_user().uid
+        return data
+
+    def get(self) -> "Member":
+        """
+        Reloads the members information from the server.
+
+        This method sends a GraphQL query to the server to fetch the latest information
+        about the members, The fetched
+        information is then used to update the corresponding attributes of the `Member` object.
+
+        Returns:
+            Member: The updated `Member` object.
+
+        Raises:
+            ResourceNotFoundError: If the query fails to fetch the member information.
+        """
+
+        if not self.id:
+            raise ValueError("Id is required")
+        query = """
+            query GetMemberReworkPyApi($userId: ID!) {
+                user(where: { id: $userId }) {
+                id
+                updatedAt
+                createdAt
+                name
+                nickname
+                email
+                defaultRole {
+                    id
+                    name
+                }
+                picture
+                isViewer
+                isExternalUser
+                canAccessAllProjects
+                accessibleProjectCount
+                userGroups {
+                    nodes {
+                        id
+                    }
+                }
+                __typename
+                }
+                projectMemberships(userId: $userId) {
+                    role {
+                        id
+                        name
+                    }
+                    project {
+                        id
+                    }
+                }
+            }
+        """
+        params = {"userId": self.id}
+
+        result = self.client.execute(query, params, experimental=True)
+        if not result:
+            raise ResourceNotFoundError(
+                message="Failed to find user as user does not exist"
+            )
```

Comment:
i don't think this is needed here. `client.execute` already handles `ResourceNotFoundError`. In such case i think this code is simply not reachable

Reply:
Fixed